0000-experiment-dataframe.md

```diff
@@ -0,0 +1,320 @@
+- quality (str): Same as `AnalysisResultData.quality`. Systematically evaluated quality of the fitting to yield this data.
+- device_components (List[DeviceComponent]): Same as `AnalysisResultData.device_components`. Usually `.physical_qubits` of the experiment.
+- extra (Dict): Same as `AnalysisResultData.result_data["_extra"]`. Free-form dictionary for analysis metadata.
+- experiment_id (str): Same as `AnalysisResultData.experiment_id`. The ID of the associated experiment.
+- verified (bool): Same as `AnalysisResultData.verified`. A boolean value to indicate if this entry is checked by human experimentalist. This can be removed since this is rarely used.
+- tags (List[str]): Same as `AnalysisResultData.tags`. An arbitrary strings to help filtering this entry.
+- source (Dict): Same as `AnalysisResultData.result_data["_source"]`. Version information of QE that was used to generate this entry.
+- created_in_db (bool): Same as `AnalysisResultData._created_in_db`. A boolean value to indicate wether this is loaded from the database or just created locally.
+- result_id (str): Same as `AnalysisResultData.result_id`. A unique ID of this entry.
+- backend (str): Same as `AnalysisResultData.backend_name`. Name of the associated backend.
+- creation_datetime (datetime): Same as `AnalysisResultData.creation_datetime`. A date time at which the experiment data received the job results.
+- experiment (str): A string representation for the associated experiment for filtering.
+
+Each table row corresponding to a single analysis result entry, i.e. `AnalysisResultData` equivalent.
+The columns must contain all information that `AnalysisResultData` provides for the backward compatibility.
+Index value is a truncated result ID which is guaranteed to be unique in the table, such as `17636b47`. We can use this ID to get a particular analysis result entry from the table.
+Some data field might be hidden from the user view.
+
+For backward compatibility, we add the `dataframe=True` option to `ExperimentData.analysis_results()` method. This options is `False` by default in the first release, and it will be `True` in future versions and eventually deprecated and removed. When `dataframe=False`, the experiment data internally generates the legacy `AnalysisResult` object from each table row.
+
+#### Example 1: Parallel experiment.
+
+```python
+pexp = ParallelExperiment(
+  [
+    T1(qubit=0, delays=delays),
+    T1(qubit=1, delays=delays),    
+  ],
+  backend=ibm_washington,
+)
+experment_data = pexp.run()
+
+experiment_data.analysis_results(dataframe=True)
+```
+
+Analysis results:
+
+|          | name   | value  | quality | device_components |  backend        | creation_datetime          | experiment | ... |
+|----------|--------|--------|---------|-------------------|-----------------|----------------------------|------------|-----|
+| 03fe91s1 | T1     | 100e-6 | good    | [Q0]              |  ibm_washington | 2022-11-22 03:55:30.313872 | T1         | ... |
```

Comment:
Where do variances/standard errors get shown get shown? As part of the value?

Reply:
Yes. This not a serialized version, and raw UFloat object will come here. So variance will be shown through its repr.